applications/luci-app-mwan3/htdocs/luci-static/resources/view/mwan3/network/interface.js

```diff
@@ -0,0 +1,285 @@
+		o.default = 'ping';
+		o.value('ping');
+		if (stats[0].type === 'file') {
+			o.value('httping');
+		}
+		if (stats[1].type === 'file') {
+			o.value('nping-tcp');
+			o.value('nping-udp');
+			o.value('nping-icmp');
+			o.value('nping-arp');
+		}
+		if (stats[2].type === 'file') {
+			o.value('arping');
+		}
+
+		o = s.option(form.Flag, 'httping_ssl', _('Enable ssl tracking'),
+			_('Enables https tracking on ssl port 443'));
+		o.depends('track_method', 'httping');
+		o.rmempty = false;
+		o.modalonly = true;
+
+		o = s.option(form.Value, 'reliability', _('Tracking reliability'),
+			_('Acceptable values: 1-100. This many Tracking IP addresses must respond for the link to be deemed up'));
+		o.datatype = 'range(1, 100)';
+		o.default = '1';
+
+		o = s.option(form.ListValue, 'count', _('Ping count'));
+		o.default = '1';
+		o.value('1');
+		o.value('2');
+		o.value('3');
+		o.value('4');
+		o.value('5');
+		o.modalonly = true;
+
+		o = s.option(form.Value, 'size', _('Ping size'));
+		o.default = '56';
+		o.depends('track_method', 'ping');
+		o.value('8');
+		o.value('24');
+		o.value('56');
+		o.value('120');
+		o.value('248');
+		o.value('504');
+		o.value('1016');
+		o.value('1472');
+		o.value('2040');
+		o.datatype = 'range(1, 65507)';
+		o.modalonly = true;
+
+		o =s.option(form.Value, 'max_ttl', _('Max TTL'));
+		o.default = '60';
+		o.depends('track_method', 'ping');
+		o.value('10');
+		o.value('20');
+		o.value('30');
+		o.value('40');
+		o.value('50');
+		o.value('60');
+		o.value('70');
+		o.datatype = 'range(1, 255)';
+		o.modalonly = true;
+
+		o = s.option(form.Flag, 'check_quality', _('Check link quality'));
+		o.depends('track_method', 'ping');
+		o.default = false;
+		o.modalonly = true;
+
+		o = s.option(form.Value, 'failure_latency', _('Failure latency [ms]'));
+		o.depends('check_quality', 1);
+		o.default = '1000';
+		o.value('25');
+		o.value('50');
+		o.value('75');
+		o.value('100');
+		o.value('150');
+		o.value('200');
+		o.value('250');
+		o.value('300');
+		o.modalonly = true;
+
+		o = s.option(form.Value, 'failure_loss', _('Failure packet loss [%]'));
+		o.depends('check_quality', 1);
+		o.default = '40';
+		o.value('2');
+		o.value('5');
+		o.value('10');
+		o.value('20');
+		o.value('25');
+		o.modalonly = true;
+
+		o = s.option(form.Value, 'recovery_latency', _('Recovery latency [ms]'));
+		o.depends('check_quality', 1);
+		o.default = '500';
+		o.value('25');
+		o.value('50');
+		o.value('75');
+		o.value('100');
+		o.value('150');
+		o.value('200');
+		o.value('250');
+		o.value('300');
+		o.modalonly = true;
+
+		o = s.option(form.Value, 'recovery_loss', _('Recovery packet loss [%]'));
+		o.depends('check_quality', 1);
+		o.default = '10';
+		o.value('2');
+		o.value('5');
+		o.value('10');
+		o.value('20');
+		o.value('25');
+		o.modalonly = true;
+
+		o = s.option(form.ListValue, "timeout", _("Ping timeout"));
+		o.default = '4';
+		o.value('1', _('%d second').format('1'));
```

Comment:
Use `N_(123, '%d second', '%d seconds')` here to allow translators to use the correct plural form for their respective languages